overlord/standby/standby.go

```diff
@@ -0,0 +1,104 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+/*
+ * Copyright (C) 2018 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package standby
+
+import (
+	"github.com/snapcore/snapd/overlord/state"
+	"time"
+)
+
+var standbyWait = 5 * time.Second
+
+type Opinionator interface {
+	CanStandby() bool
+}
+
+// StandbyHelper tracks if snapd can go into socket activation mode
+type StandbyHelper struct {
```

Comment:
This one might be StandbyOpinions.. "Helper" is generally too vague to be.. well.. helpful, ironically :-)